### DISCUSSION TITLE
Questions about planting a git lawn

### DISCUSSION THREAD

**Discussion by CoderJDan** [community/community#147156] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![image](https://github.com/user-attachments/assets/d7c53bee-fac7-4b99-9fcc-963f69710ed2)
![image](https://github.com/user-attachments/assets/30ccc88c-6811-4e4f-a0f0-a40eef6d5934)

hello. I manage my lawn planting via git commit. I was modifying the README file for one project and ended up doing too many commits, which caused the commit count to go up by leaps and bounds, so I cleared the recent commit history from the local repository and forced a push back. My question is, I successfully cleared the commit history, but the git commit sod still has that information. How can I clear it?

**Comment by Onek8** [comment]
![image](https://github.com/user-attachments/assets/cfe4f3ff-7a0f-4f3e-b51c-e70c934f4529)
Did you do the following?

**Reply by Onek8** [reply]
Thread source at https://stackoverflow.com/questions/13716658/how-to-delete-all-commit-history-in-github
